ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusEditMessage.qml

```diff
@@ -59,7 +59,15 @@ Item {
                 size: StatusBaseButton.Size.Small
                 enabled: !!chatInputLoader.item && chatInputLoader.item.messageText.trim().length > 0
                 onClicked: {
-                    editCompleted(!chatInputLoader.item ? "" : chatInputLoader.item.messageText)
+                    let text = ""
+                    if (chatInputLoader.item) {
+                        if (chatInputLoader.item.getTextWithPublicKeys) {
```

Comment:
```suggestion
                        if (chatInputLoader.item.getTextWithPublicKeys()) {
```

parens missing?

Reply:
No, I'm just testing if the function exists on the component. If I added parens and the function didn't exist, it would throw.